en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,274 @@
+Manage assets with grunt
+========================
+
+Context
+-------
+
+To automate all the assets management process, the Open Orchestra Back Office uses the task manager
+`Grunt`_.
+
+*Grunt* is looking for its configuration inside the ``Gruntfile.js`` file in the root directory of
+the project. This file mainly parses external configuration files, split in grunt tasks and grunt
+tasks options.
+
+In order to simplify the management of external bundles assets, each bundle describes its own
+grunt task and/or task options. As a result, adding a new bundle will simply require a little change
+in the ``Gruntfile.js`` of your project to load the specific configuration.
+
+In this documentation, you will see:
+
+- how to `configure your project to use grunt`_ 
+- how to `add an external bundle using grunt`_ tasks / options
+- how to `add your specific tasks / options to your bundles`_ 
+
+.. _configure your project to use grunt:
+
+Configuring your project to use grunt
+-------------------------------------
+
+When you install your Open Orchestra Back Office, you have to create a ``Gruntfile.js`` to manage
+the assets included in the bundles you choose. This file loads and parses configuration files to
+produce the grunt configuration.
+
+Here is a sample providing standard functionnalities.
```

Comment:
Here is a nice place to put the part about the configuration being split between bundles.
Also, the Gruntfile you provide is actually the one from orchestra that the users should actually use, so i would make this aspect more explicit (rather than introducing it like a "sample with standard functionnalities")